### DISCUSSION TITLE
how do i resolve this module package conflict / issue?

### DISCUSSION THREAD

**Discussion by kolinfluence** [etcd-io/etcd#15967] [Q&A]
how does this work? isnt this a problem?
```
pkg/models/etcd imports
        github.com/etcd-io/etcd imports
        github.com/coreos/etcd/etcdmain imports
        github.com/coreos/etcd/etcdserver imports
        github.com/coreos/etcd/mvcc/backend imports
        github.com/coreos/bbolt: github.com/coreos/bbolt@v1.3.7: parsing go.mod:
        module declares its path as: go.etcd.io/bbolt
                but was required as: github.com/coreos/bbolt
```

**Comment by ahrtr** [comment]
All `github.com/coreos/*` items should NOT be in the dependency chain at all. 

Please provide the exact steps you executed.

**Reply by jmhbnz** [reply]
Related duplicate issue with some additional context: https://github.com/etcd-io/etcd/issues/15968

**Comment by kolinfluence** [comment]
this is a lot of work.
try compiling and using 
https://github.com/CodisLabs/codis

for using etcd
the problem's there
